src/bp/ui-admin/src/telemetry/telemetry_type.d.ts

```diff
@@ -0,0 +1,34 @@
+export interface StoreInfoBody {
+  storedInfo: string
+  loadInfo: Function
+}
+
+export interface StoreInfoType {
+  [key: string]: StoreInfoBody
+}
+
+export interface EventPackageInfoType {
+  [key: string]: EventPackageInfoBody
+}
+
+export interface EventPackageInfoBody {
```

Comment:
this can be nested in parent type as its not used anywhere, see example below

Reply:
Fixed in : e135fcf